sanic/request.py

```diff
@@ -200,21 +203,73 @@ def files(self):
 
         return self.parsed_files
 
-    @property
-    def args(self):
-        if self.parsed_args is None:
+    def get_args(
```

Comment:
This is a major breaking change for someone who is already using this feature. Instead of removing it directly, should go via the deprecate -> remove phase. It can be a major pain otherwise for anyone trying to upgrade to a latest version

Reply:
Nothing has changed for who is using `args` property. Look below -
```python
args = property(get_args)
```

Reply:
@Amanit My bad. Sorry for the confusion. I totally missed that line.